engine/engine.go

```diff
@@ -31,6 +33,8 @@ type EngineAPI struct {
 	builder                  *builder.Builder
 	txValidator              TxValidator
 	blockStore               DB
+	appchainClient           *appchainClient.Context
```

Comment:
Probably: should remove `appchainClient` as a member of the engine, since it gets pass on construction as a member of the signer.

Reply:
e066e53